.storybook/main.js

```diff
@@ -4,7 +4,7 @@ const path = require("path");
 
 module.exports = {
   stories: ["../stories/**/*.stories.js"],
-  addons: ["@storybook/addon-knobs"],
+  addons: ["@storybook/addon-controls"],
```

Comment:
<img width="883" alt="CleanShot 2023-10-26 at 13 30 05@2x" src="https://github.com/mozilla/glean-dictionary/assets/28797553/fed343df-81f2-47a7-b8e0-748167be9616">

Looking at the error logs, seems like we also need to install the `"@storybook/addon-docs"` dependency and add it here for the autogenerated docs to work.

```suggestion
  addons: ["@storybook/addon-controls", "@storybook/addon-docs"],
```